dsl-reference.md

```diff
@@ -368,7 +369,7 @@ The [HTTP Call](#http-call) enables workflows to interact with external services
 | Name | Type | Required | Description|
 |:--|:---:|:---:|:---|
 | method | `string` | `yes` | The HTTP request method. |
-| endpoint | [`endpoint`](#endpoint) | `yes` | An URI or an object that describes the HTTP endpoint to call. |
+| endpoint | `string`\|[`endpoint`](#endpoint) | `yes` | An URI or an object that describes the HTTP endpoint to call. |
```

Comment:
There is an `/` escape character here that afaik is useless and should therefore be removed

Reply:
it's useful or otherwise it "brakes" the table.

| Name | Type | Required | Description|
|:--|:---:|:---:|:---|
| endpoint | `string`\|[`endpoint`](#endpoint) | `yes` | An URI or an object that describes the HTTP endpoint to call. |

vs

| Name | Type | Required | Description|
|:--|:---:|:---:|:---|
| endpoint | `string`|[`endpoint`](#endpoint) | `yes` | An URI or an object that describes the HTTP endpoint to call. |

Reply:
Brakes because otherwise the table goes too fast? 😂
Sorry, I couldn't resist, I'm out.